src/arr/trove/statistics.arr

```diff
@@ -242,6 +243,11 @@ fun multiple-regression(x_s_s :: List<List<Number>>, y_s :: List<Number>) -> (Li
   B_pred_fn
 end
 
+fun multiple-regression(x_s_s :: List<Any>, y_s :: List<Number>) -> (Any -> Number):
```

Comment:
The `Any`s in this signature are wrong, correct?  They should both be `List<Number>`?